ingest/cell_metadata.py

```diff
@@ -105,8 +110,8 @@ def validate_header_keyword(self):
             if self.headers[0] != 'NAME':
```

Comment:
Do we need to worry about case sensitivity here?

Reply:
No, the parent if statement checks
` if self.headers[0].casefold() == 'NAME'.casefold()`
that particular line is used to trigger a warning (rather than an error) that the expected official format using NAME has been provided in the wrong case.